manuscript/simulators.tex

```diff
@@ -0,0 +1,27 @@
+\section{Fuel Cycle Simulators}
+
+\subsection{\gls{VISION}}
+\gls{VISION} is a fuel cycle simulation
+that is intended to evaluate "what if" scenarios.
+It considers the nuclear fuel cycle at a systems level,
+not in individual facilities \cite{jacobson_user_2011}.
+It tracks materials in different steps of the fuel cycle in an isotopic level,
```

Comment:
correction, you can now ask vision not to lump fission products in the way it used to.